Libraries/Components/ScrollResponder.js

```diff
@@ -112,6 +113,15 @@ type State = {
 };
 type Event = Object;
 
+function isTagInstanceOfTextInput(tag) {
+  var instance = getInstanceFromNode(tag);
+  return instance && (
+    instance.constructor.displayName === 'AndroidTextInput' ||
```

Comment:
http://stackoverflow.com/questions/27824548/comparing-two-components-is-component-x-an-instance-of-component-a